config/katello-answers.yaml

```diff
@@ -65,7 +65,7 @@ foreman::plugin::puppetdb: false
 foreman::plugin::remote_execution: false
 foreman::plugin::setup: false
 foreman::plugin::tasks: true
-foreman::plugin::templates: false
+foreman::plugin::templates: true
```

Comment:
This is the Foreman templates plugin, but the Redmine issue mention `--foreman-proxy-templates` which is the smart proxy plugin.